README.md

```diff
@@ -33,6 +33,26 @@ Or alternatively, without `docker compose`:
 docker run --rm -it --network host --volume ./opt:/opt/famedly-sync-agent docker-oss.nexus.famedly.de/famedly-sync-agent:latest
 ```
 
+### Configuration
+
+The ldap-sync expects a configuration file located at `./config.yaml`. The default path can be changed setting the new path to the environment variable `FAMEDLY_LDAP_SYNC_CONFIG`.
+
+Also, individual configuration items and the whole configuration can be set using environment variables. For example, the following configuration could be set like this:
+
+Configuration yaml
+```yaml
+ldap:
+          url: ldap://localhost:1389
```

Comment:
I'm sorry, but 8 space tabs look ugly, we mostly use 2: [.editorconfig:12](https://github.com/famedly/ldap-sync/blob/9dfabce2a41e2749713e545aeee716810574557f/.editorconfig#L12)